runtime/integration-tests/src/liquidity_pools/pallet/development/transfers.rs

```diff
@@ -0,0 +1,250 @@
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+use cfg_primitives::{constants::currency_decimals, parachains, Balance};
+use cfg_types::{
+	tokens::{CrossChainTransferability, CurrencyId, CustomMetadata},
+	xcm::XcmMetadata,
+};
+use development_runtime::{Balances, OrmlAssetRegistry, OrmlTokens, RuntimeOrigin, XTokens};
+use frame_support::assert_ok;
+use orml_traits::{asset_registry::AssetMetadata, FixedConversionRateProvider, MultiCurrency};
+use runtime_common::{
+	xcm::general_key,
+	xcm_fees::{default_per_second, ksm_per_second},
+};
+use sp_runtime::traits::BadOrigin;
+use xcm::{
+	latest::{Junction, Junction::*, Junctions::*, MultiLocation, NetworkId, WeightLimit},
+	VersionedMultiLocation,
+};
+use xcm_emulator::TestExt;
+
+use crate::{
+	liquidity_pools::pallet::{
+		development::{
+			setup::{
+				centrifuge_account, cfg, moonbeam_account, ALICE, BOB, CHARLIE, PARA_ID_MOONBEAM,
+			},
+			test_net::{Development, Moonbeam, RelayChain, TestNet},
+			tests::register_ausd,
+		},
+		xcm_metadata,
+	},
+	utils::AUSD_CURRENCY_ID,
+};
+
+/*
+
+NOTE: We hardcode the expected balances after an XCM operation given that the weights involved in
+XCM execution often change slightly with each Polkadot update. We could simply test that the final
+balance after some XCM operation is `initialBalance - amount - fee`, which would mean we would
+never have to touch the tests again. However, by hard-coding these values we are forced to catch
+an unexpectedly big change that would have a big impact on the weights and fees and thus balances,
+which would go unnoticed and untreated otherwise.
+
+ */
+
+#[test]
+fn transfer_cfg_to_sibling() {
+	TestNet::reset();
+
+	let alice_initial_balance = cfg(10_000);
+	let bob_initial_balance = cfg(10_000);
+	let transfer_amount = cfg(5);
+	let cfg_in_sibling = CurrencyId::ForeignAsset(12);
+
+	// CFG Metadata
+	let meta: AssetMetadata<Balance, CustomMetadata> = AssetMetadata {
+		decimals: 18,
+		name: "Development".into(),
+		symbol: "CFG".into(),
+		existential_deposit: 1_000_000_000_000,
+		location: Some(VersionedMultiLocation::V3(MultiLocation::new(
+			1,
+			X2(
+				Parachain(parachains::polkadot::centrifuge::ID),
+				general_key(parachains::polkadot::centrifuge::CFG_KEY),
+			),
+		))),
+		additional: CustomMetadata {
+			transferability: CrossChainTransferability::Xcm(Default::default()),
+			..CustomMetadata::default()
+		},
+	};
+
+	Development::execute_with(|| {
+		assert_eq!(Balances::free_balance(&ALICE.into()), alice_initial_balance);
+		assert_eq!(Balances::free_balance(&moonbeam_account()), 0);
+
+		assert_ok!(OrmlAssetRegistry::register_asset(
+			RuntimeOrigin::root(),
+			meta.clone(),
+			Some(CurrencyId::Native),
+		));
+	});
+
+	Moonbeam::execute_with(|| {
+		assert_eq!(OrmlTokens::free_balance(cfg_in_sibling, &BOB.into()), 0);
```

Comment:
```suggestion
		assert_eq!(Tokens::free_balance(cfg_in_sibling, &BOB.into()), 0);
```

I think we should get used to using `Tokens` everywhere instead of `OrmlTokens`. But just for the future. Need to change to many tests for getting this cleanly in now.